src/ucp/core/ucp_ep.c

```diff
@@ -1835,6 +1835,69 @@ ucp_lane_index_t ucp_ep_lookup_lane(ucp_ep_h ucp_ep, uct_ep_h uct_ep)
     return UCP_NULL_LANE;
 }
 
+static void
+ucp_ep_init_is_connected_params(uct_ep_is_connected_params_t *params,
```

Comment:
Maybe rename to `fill` or something similar.
The func assigns values to a "params" struct, rather then "initializing" some state.